Makefile.pre.in

```diff
@@ -2759,6 +2760,12 @@ patchcheck: all
 check-limited-abi: all
 	$(RUNSHARED) ./$(BUILDPYTHON) $(srcdir)/Tools/build/stable_abi.py --all $(srcdir)/Misc/stable_abi.toml
 
+.PHONY: bundle-ensurepip-dists
+bundle-ensurepip-dists:
+	if test "x$(ENSUREPIP)" != "xno"  ; then \
+		$(RUNSHARED) $(PYTHON_FOR_BUILD) -m ensurepip.bundle -v ; \
```

Comment:
:exclamation: I need help with this — my attempts to integrate the bundler into the build process fail in some CI jobs (supposedly because of the `./configure` CLI flags).